span-normalizer/src/test/java/org/hypertrace/core/spannormalizer/SpanNormalizerTest.java

```diff
@@ -50,27 +49,25 @@ public void whenJaegerSpansAreProcessedExpectRawSpansToBeOutput() {
         (Function<Entry<String, ConfigValue>, Object>) Entry::getKey,
         (Function<Entry<String, ConfigValue>, Object>) Entry::getValue));
 
-    Map<String, Object> baseProps = underTest.getBaseStreamsConfig();
-    Map<String, Object> streamsProps = underTest.getStreamsConfig(config);
-    baseProps.forEach(streamsProps::put);
-    Map<String, Object> mergedProps = streamsProps;
-    streamsProps.put(SpanNormalizerConstants.SPAN_NORMALIZER_JOB_CONFIG, config);
+    Map<String, Object> mergedProps = new HashMap<>();
+    underTest.getBaseStreamsConfig().forEach(mergedProps::put);
+    underTest.getStreamsConfig(config).forEach(mergedProps::put);
+    mergedProps.put(SpanNormalizerConstants.SPAN_NORMALIZER_JOB_CONFIG, config);
+
     StreamsBuilder streamsBuilder = underTest
         .buildTopology(mergedProps, new StreamsBuilder(), new HashMap<>());
 
     Properties props = new Properties();
-    streamsProps.forEach(props::put);
+    mergedProps.forEach(props::put);
 
     TopologyTestDriver td = new TopologyTestDriver(streamsBuilder.build(), props);
     TestInputTopic<byte[], Span> inputTopic = td
         .createInputTopic(config.getString(SpanNormalizerConstants.INPUT_TOPIC_CONFIG_KEY),
             Serdes.ByteArray().serializer(), new JaegerSpanSerde().serializer());
 
     // make sure the schema reistry url starts with 'mock://' including the one specified in application.conf
```

Comment:
I think this comment is not anymore valid?

Reply:
Yes. Fixed.